internal/command/org_test.go

```diff
@@ -456,6 +456,158 @@ func TestCommandSide_AddOrg(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "add org (remove invalid characters), no error",
```

Comment:
"no error" can be ignored, as sanitization will never result in an error. There are a lot more possible combinations but that can be handled in unit tests specific for the sanitization.

Reply:
Done!